components/gateway/internal/auditlog/client.go

```diff
@@ -0,0 +1,148 @@
+package auditlog
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/kyma-incubator/compass/components/gateway/internal/auditlog/model"
+
+	"github.com/pkg/errors"
+)
+
+const LogFormatDate = "2006-01-02T15:04:05.999Z"
+
+const ConfigChangeURLPath = "/audit-log/v2/configuration-changes"
+const SecurityEventURLPath = "/audit-log/v2/security-events"
+
+//go:generate mockery -name=UUIDService -output=automock -outpkg=automock -case=underscore
+type UUIDService interface {
+	Generate() string
+}
+
+//go:generate mockery -name=TimeService -output=automock -outpkg=automock -case=underscore
+type TimeService interface {
+	Now() time.Time
+}
+
+type AuditlogConfig struct {
```

Comment:
What is going to happen if there's no audit log service? Please make it possible for the gateway to work without it